docs/pages/admin-guides/teleport-policy/integrations/aws-sync.mdx

```diff
@@ -155,6 +155,7 @@ The IAM policy includes the following directives:
         "s3:GetBucketPolicy",
         "s3:ListBucket",
         "s3:GetBucketLocation",
+        "s3:GetBucketTagging",
```

Comment:
The changelog should probably mention something about this new requirement, otherwise self-hosted users won't know they need to update their permissions.

Reply:
I don't think this is yet being used but added a note to changelog.